frappe/public/scss/desk/sidebar.scss

```diff
@@ -104,10 +104,10 @@ body[data-route^="Module"] .main-menu {
 	}
 
 	.sidebar-image-section {
+		width: min(100%, 170px);
```

Comment:
Need capital M here to avoid libsass issue
https://github.com/frappe/frappe/runs/5198762120?check_suite_focus=true#step:13:214